### ISSUE TITLE
 Could not connect to freevpn.ninja

### ISSUE THREAD

**Issue by szefkamil** [shadow-1/yi-hack-v3#176]
Connecting to freevpn.ninja (185.53.179.7:443)
wget: TLS error from peer (alert code 40): handshake failure
wget: error getting response: Connection reset by peer

Return Code: 1

I have this problem when i want to downlad proxy list because few days ago camera is offline.

**Comment by adrianwy** [comment]
I have the same issue. Could you please advice what we can do?

**Comment by adrianwy** [comment]
I have found the https://github.com/shadow-1/yi-hack-v3/issues/135 but non of them are working

**Comment by szefkamil** [comment]
http://spys.one/proxys/CN/

go to this website and tap manualy

random_chain
chain_len = 1
tcp_read_time_out 15000
tcp_connect_time_out 8000
[ProxyList]
socks5 223.241.214.44 1217
socks5 211.95.255.42 1080
socks5 118.89.231.84 30687
socks5 119.29.163.54 1080
socks5 61.145.213.14 1080
socks5 218.58.210.233 1080
socks5 221.209.62.170 1080

in system config sheet

![obraz](https://user-images.githubusercontent.com/26625926/38576759-5b2569fe-3cff-11e8-82fe-065fa59e4302.png)

**Comment by szefkamil** [comment]
![obraz](https://user-images.githubusercontent.com/26625926/38576816-7d6ffef2-3cff-11e8-8597-3e7c37ccd525.png)

please use this and tap manualy socks5 and ip server please remeber about dots between numbers for example [socks5 211.95.255.42 1080]

**Comment by adrianwy** [comment]
Yes that is the same link. I have just put all the Proxy on the configuration file and tested but non of them are working :(

**Comment by szefkamil** [comment]
please try few of them maybe this one


221.232.233.159:1080 | SOCKS5

**Comment by szefkamil** [comment]
after that please reboot camera by 
![obraz](https://user-images.githubusercontent.com/26625926/38576981-f7174080-3cff-11e8-99d6-1328d3a7b13a.png)

**Comment by szefkamil** [comment]
and try to use app in smartphone and everything should be okay

**Comment by szefkamil** [comment]
wait for blue led inform that wifi connection is okay in chinese

**Comment by szefkamil** [comment]
remove other proxy leave only this one

**Comment by adrianwy** [comment]
Ok will do and I will let you know, thanks Kamil

**Comment by szefkamil** [comment]
no problem, i want to help you
i spend last sunday morning on this and have a lot problems too

**Comment by szefkamil** [comment]
please sent me a picture of your website and i will check
and we wil try to resolve it together

**Comment by szefkamil** [comment]
random_chain

chain_len = 1

tcp_read_time_out 15000
tcp_connect_time_out 8000

[ProxyList]
socks5 218.72.111.83 1080
socks5 114.67.67.67 1080
socks5 183.159.92.171 1080
socks5 117.64.50.173 1080

**Comment by szefkamil** [comment]
this is my setup

**Comment by szefkamil** [comment]
any progress??

**Comment by adrianwy** [comment]
Did you check the 'Check Proxy List'? Does it work for you? Non of them are working

**Comment by szefkamil** [comment]
no I didn't check this

**Comment by szefkamil** [comment]
only write proxy from this web and reboot camera

**Comment by szefkamil** [comment]
put phone to scan qr code and wait for wifi connection
and click next on the app

**Comment by adrianwy** [comment]
Which version do you have? Chinese or international?

**Comment by szefkamil** [comment]
chinese

**Comment by razorj** [comment]
I have the same issue, which was solved by repeatedly trying more Proxy a few days ago. However, I just found the problem occurs again, so I would assume the proxy is not valid anymore. Is there solution can last longer? How to make a automated proxy list just like old service by freevpn.ninja. 

By the way, mine is also Chinese version. Thx a lot!

**Comment by szefkamil** [comment]
Hello all,
Yesterday i check this metod and it's working fine:

1. download app from google play store: kibou VPN 
2. copy few proxy adress to proxychains-ng tab
3. reboot camera
4. camera is working and connecting.

Reagards
Kamil

**Comment by szefkamil** [comment]
other method is to use vpn spys.one/free-proxy-list/CN
copy few adresses with ratio near to 100% and it's working.

I checked also

**Comment by adrianwy** [comment]
Ok thanks Kamil for your support. I will try to manage it this evening

**Comment by szefkamil** [comment]
My pleasure

czw., 12.04.2018, 21:37 użytkownik adrianwy <notifications@github.com>
napisał:

**Comment by williamwl1215** [comment]
sorry i still couldnt connected the proxy server, no matter how many proxy server address which are 100% uptime i copy,  when test config server connectivity still failed.  reboot can not solve the problem either. 

is there anything wrong with my config? 

such as last time i test the proxy 
HTTP 101.200.56.177:3128	
SOCKS5 120.79.78.178:1080	
SOCKS5 114.99.70.111:1080	
 and still always show me like this
![qq](https://user-images.githubusercontent.com/37769732/38713297-0e157838-3f03-11e8-8073-e3e89272eae7.png)

**Comment by szefkamil** [comment]
Remove : before port example
Socks5 192.192.192 1080


pt., 13.04.2018, 04:12 użytkownik williamwl1215 <notifications@github.com>
napisał:

**Comment by nelson2mts** [comment]
szefkamil still cannot connect sametime can sametime cannot pls help me tq.

**Comment by strelokr** [comment]
new list
socks5 149.129.107.164 1080
socks5 118.193.190.35 8088
socks5 103.115.44.150 1080 
socks5 192.151.246.90 3128

**Comment by tomnyw** [comment]
Hi there 
I have tried all the proxy servers from http://spys.one/free-proxy-list/CN/ and i cant seem to connect. 
is there any way to go about this ?

**Comment by szefkamil** [comment]
Fuck this camera and we should buy a new unlocked in US or EU

czw., 14 mar 2019, 16:31 użytkownik tomnyw <notifications@github.com>
napisał:
